services/provider/server/server.go

```diff
@@ -664,6 +672,14 @@ func (s *OCSProviderServer) GetStorageClaimConfig(ctx context.Context, req *pb.S
 					"clusterID": getSubVolumeGroupClusterID(subVolumeGroup),
 					"csi.storage.k8s.io/snapshotter-secret-name": provisionerSecretName,
 				})})
+
+			extR = append(extR, &pb.ExternalResource{
+				Name: "cephfs",
+				Kind: "VolumeGroupSnapshotClass",
+				Data: mustMarshal(map[string]string{
+					"clusterID": getSubVolumeGroupClusterID(subVolumeGroup),
```

Comment:
Can we please calculate the cluster-ID once, store it in a var, and use it where necessary?